client/state/persistent-counter/actions.ts

```diff
@@ -12,7 +12,7 @@ export const incrementCounter =
 			return;
 		}
 
-		dispatch(
+		return dispatch(
```

Comment:
This function is used in only one place:
https://github.com/Automattic/wp-calypso/blob/2b29508bb14ead48af36adb4d660a57195d1e7f6/client/my-sites/scan/main.tsx#L74
IMO, adding the `return` should not affect this usage.